README.md

```diff
@@ -13,6 +13,22 @@ Homepage: [http://lampwww.epfl.ch/~hmiller/f-p](http://lampwww.epfl.ch/~hmiller/
 Examples and tests can be launched from the sbt prompt:
 
 ```
+> project samples
+> [info] Set current project to f-p samples (in build file:/Users/nrm/Sources/nrm/bd/f-p/)
```

Comment:
Perhaps replace paths that are valid only locally with "..."? Like this: `(in build file:/.../f-p/)`.